docs/stack/alchemy.md

```diff
@@ -0,0 +1,7 @@
+# Alchemy
+
+Arc.js library facilitates development of "dApps" (Decentralized applications) for interacting with DAOs.
+
+**DAOstack has built its own dApp called [Alchemy](https://alchemy.daostack.io/), a front-end interface for DAOs**, or more specifically, for budget management in decentralized organizations. Alchemy allows end users to make collaborative budgeting decisions and allocations using the [Holographic Consensus protocol](https://www.youtube.com/watch?v=1De0MoStSkY&feature=youtu.be&t=11m50s).
```

Comment:
`collaborative budgeting decisions and Ethereum blockchain interaction of any kind`